samples/highcharts/chartchooser/continuous-comparison-line-dash/demo.js

```diff
@@ -0,0 +1,47 @@
+Highcharts.chart("container", {
+    chart: {
+        type: "spline"
+    },
+    title: {
+        text: "Operating Systems Trends"
+    },
+    subtitle: {
+        text:
+        'Source: <a href="https://trends.google.com/trends/explore?date=all&q=%2Fm%2F02wxtgw,%2Fm%2F03wbl14,%2Fm%2F04r_8,%2Fm%2F0fpzzp,%2Fm%2F055yr">Google Trends </a>'
+    },
+
+    yAxis: {
+        max: 100,
+        title: {
+            enabled: false
+        }
+    },
+    tooltip: {
+        shared: true,
+        xDateFormat: "%Y-%m",
+        crosshairs: true,
+        useHTML: true,
+        headerFormat: '<table><tr><th colspan="2">{point.key}</th></tr>',
+        pointFormat:
+        "<tr><td>{series.name} </td>" +
+        '<td style="text-align: right"><b>{point.y}</b></td></tr>',
+        footerFormat: "</table>"
+    },
+
+    data: {
+        csvURL:
+        "https://raw.githubusercontent.com/mekhatria/demo_highcharts/master/operatingSystemsTrends.csv"
```

Comment:
Do not link to resources on your personal profile. Put it in /samples/data and link it through jsdelivr. Search for `/samples/data` in the samples folder to find use cases.

Reply:
Done 👍

Reply:
After the change the `cvsURL` is https://cdn.jsdelivr.net/gh/highcharts/highcharts@v7.0.0/samples/data/operating-systems-trends.csv . This works in our utils because of a rewrite, but it will not work online because that file does not exist as of v7.0.0. The trick is to first make a commit with the data file (which you already did), then you can refer to that commit. This result is https://cdn.jsdelivr.net/gh/highcharts/highcharts@1e340a13/samples/data/operating-systems-trends.csv , and we can confirm that that file is online.